plugins/modules/packaging/os/pacman.py

```diff
@@ -19,6 +21,7 @@
     - Indrajit Raychaudhuri (@indrajitr)
     - Aaron Bull Schaefer (@elasticdog) <aaron@elasticdog.com>
     - Maxime de Roucy (@tchernomax)
+    - Jean Raby (@jraby)
```

Comment:
You need to add yourself also to `.github/BOTMETA.yml`'s `maintainers` list for the `$modules/packaging/os/pacman.py` entry.